app/src/main/java/com/example/drawingApp/DrawingFieldView.kt

```diff
@@ -61,6 +61,14 @@ class DrawingFieldView @JvmOverloads constructor(
         paint.color = c
     }
 
+    fun getBitmap(): Bitmap? {
```

Comment:
This could potentially be very slow and it's on the main thread here. I think a system of using a lambda callback for what to do post save and using a coroutine would be a better approach if you really want this

Reply:
this is out of scope really, lets revisit this when its in scope again.